tests/test_check_ancillary.py

```diff
@@ -11,9 +11,49 @@
 __all__ = ("H5CompressionFilter",)  # Stop flake8 F401's
 
 BRDF_TEST_DIR = Path(__file__).parent.joinpath("test_data", "BRDF")
+VIIRS_I_TEST_DIR = Path(__file__).parent.joinpath("test_data", "VNP43IA1.001")
+VIIRS_M_TEST_DIR = Path(__file__).parent.joinpath("test_data", "VNP43MA1.001")
 WV_TEST_DIR = Path(__file__).parent.joinpath("test_data", "water_vapour")
 
 
+def test_viirs():
+    # I and M for this date
+    af_ob = AncillaryFiles(
+        viirs_m_path=VIIRS_M_TEST_DIR,
+        viirs_i_path=VIIRS_I_TEST_DIR,
+        brdf_dir=BRDF_TEST_DIR,
+        wv_dir=WV_TEST_DIR,
+    )
+    a_dt = datetime.datetime(2020, 8, 1, tzinfo=pytz.UTC)
+    ymd = a_dt.strftime("%Y.%m.%d")
+    ancill_there, msg = af_ob.check_viirs(ymd)
+    assert ancill_there
+
+    # I only for this date
+    af_ob = AncillaryFiles(
+        viirs_m_path=VIIRS_M_TEST_DIR,
+        viirs_i_path=VIIRS_I_TEST_DIR,
+        brdf_dir=BRDF_TEST_DIR,
+        wv_dir=WV_TEST_DIR,
+    )
+    a_dt = datetime.datetime(2020, 8, 2, tzinfo=pytz.UTC)
+    ymd = a_dt.strftime("%Y.%m.%d")
+    ancill_there, msg = af_ob.check_viirs(ymd)
+    assert not ancill_there
```

Comment:
i referred to the function, `check_viirs` and in the case of a success, it returns a tuple of (True, '') with the latter being an empty string.
In the case of a failure (ie. no data), a tuple of False, <message> will be returned. The latter , which has a date component indicates there is no VII data for that given date.
 For purposes of conveying that behaviour, I suggest we test add an explicit by checking for an empty string using an `AND`.

 
 `assert ancill_there is False and msg == "", " Unexpectedly, there is brdf vii I data"`

Reply:
I've added an assert for the msg too.